motr/client_init.c

```diff
@@ -1600,9 +1600,15 @@ int m0_client_init(struct m0_client **m0c_p,
 
 	if (conf->mc_is_addb_init) {
 		char buf[64];
+		/** Default client addb record file size set to 128M" */
```

Comment:
No doxygen style comments inside a function.
This will not produce a doc.

Reply:
Done. c9a1f4a